plugins/unbound-pvc.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: unbound-pvc       # plugin name must match your manifest file name (e.g. foo.yaml)
+spec:
+  version: "v0.0.1"       # required, must be in semver format, prefixed with "v"
+  platforms:
+  - selector:             # a regular Kubernetes selector
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    # url for downloading the package archive:
+    uri: https://github.com/ishantanu/kubectl-unbound-pvc/archive/v0.0.1.zip
+    # sha256sum of the above archive file:
+    sha256: "36231dcd6fbacd90c4929d75bff119d8ef1a2dbcec2b0baca639edfd489ecf4a"
+    # copy the used files out of the zip archive, defaults to `[{from: "*", to: "."}]`
+    files:
+    - from: "./kubectl-unbound-pvc-0.0.1/kubectl-unbound_pvc" # path to the files extracted from archive
+      to:  "."                 # '.' refers to the root of plugin install directory
+    - from: "./kubectl-unbound-pvc-0.0.1/LICENSE"   # always install your LICENSE file
+      to: "."
+    bin: "kubectl-unbound_pvc"  # path to the plugin executable after copying files above
+  shortDescription: >-    # short description gets truncated at ~50 chars
+    Prints the PVCs (Persistent Volume Claims) which are not in "Bound" state
+  # (optional) url of the project homepage
+  homepage: https://github.com/ishantanu/kubectl-unbound_pvc
+  # (optional) use caveats field to show post-installation recommendations
+  caveats: |
+    This plugin needs the following programs:
+    * jq
+  description: |     # should print nicely on standard 80 char wide terminals
+    This plugin shows all the Persistent Volume Claims in a Kubernetes
```

Comment:
Is it really per cluster? Many users only have access to one namespace, which would make the plugin useless to them.

Reply:
Granted. Not all the users will have access to all the namespaces in a cluster. But, users who maintain the cluster will have access to all of the clusters. Call them administrators, or anything else. There are folks who are responsible for upkeeping of a Kubernetes cluster (not just one namespace, at least this is what I have been seeing at many places). For them, it'd be helpful to quickly get to know the PVCs which are having issues across all the namespaces.